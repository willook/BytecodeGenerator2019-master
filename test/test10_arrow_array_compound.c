void main () {
   int a[2];

   a[0] = 3;
   a[1] = 6;

   a -> + 3 -> * 2;
   _print(a[0]);  // 12
   _print(a[1]);  // 18

   a -> / 2 -> - 2;
   _print(a[0]);  // 4
   _print(a[1]);  // 7
}