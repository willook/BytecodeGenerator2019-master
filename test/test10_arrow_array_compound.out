12
18
4
7
